initialize/500.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Not sure why are we committing this file.

Reply:
In order to create a mosaic chain. It expects a config in a particular folder which is (initialise) with name of ${auxiliaryChain}.json name. That's why we need this file.

Reply:
Yes please, but question was more why do we need a file for auxiliaryChain:500